<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>821</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gdlyMain" rowstretch="1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>551</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="hzlyTbwResult" stretch="1">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tbwResult">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>1</number>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>0</number>
           </property>
           <property name="columnCount">
            <number>10</number>
           </property>
           <column>
            <property name="text">
             <string>公司</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>联系人</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>联系电话</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>手机</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>传真</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>邮编</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>公司主页</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>公司地址</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>备注</string>
            </property>
           </column>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QTabWidget" name="tabTasks">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabTaskRuning">
            <property name="statusTip">
             <string>Task is running</string>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <attribute name="title">
             <string>未完成</string>
            </attribute>
            <widget class="QWidget" name="horizontalLayoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>201</width>
               <height>461</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="hzlyRunningTask">
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QListWidget" name="listRunningTasks"/>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tabTaskCompleted">
            <attribute name="title">
             <string>已完成</string>
            </attribute>
            <widget class="QWidget" name="horizontalLayoutWidget_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>201</width>
               <height>461</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="hzlyCompletedTask">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <property name="topMargin">
               <number>5</number>
              </property>
              <property name="rightMargin">
               <number>5</number>
              </property>
              <property name="bottomMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QListWidget" name="listCompletedTasks"/>
              </item>
             </layout>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>821</width>
      <height>75</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="ltOutput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionTo_Excel"/>
     <addaction name="actionTo_File"/>
    </widget>
    <addaction name="actionNew_Task"/>
    <addaction name="menuExport"/>
    <addaction name="actionStop"/>
    <addaction name="actionStopAll"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>用户</string>
    </property>
    <addaction name="actionRegiste"/>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionInsert"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuUser"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionTo_Excel">
   <property name="text">
    <string>To Excel</string>
   </property>
  </action>
  <action name="actionNew_Task">
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionRegiste">
   <property name="text">
    <string>Registe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionTo_File">
   <property name="text">
    <string>To File</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionStopAll">
   <property name="text">
    <string>StopAll</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tbwResult</sender>
   <signal>cellChanged(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>itemChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabTasks</sender>
   <signal>tabBarClicked(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>tabTasksClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listCompletedTasks</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>completedTasksItemClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>itemChanged()</slot>
  <slot>tabTasksClicked()</slot>
  <slot>completedTasksItemClicked()</slot>
 </slots>
</ui>
